src/GitHub.UI.Reactive/Assets/Controls/Validation/UserErrorMessages.xaml

```diff
@@ -29,20 +29,14 @@
                             <ui:OcticonPath x:Name="icon" Icon="{TemplateBinding Icon}" Height="16" Fill="{TemplateBinding Fill}" />
                         </Viewbox>
 
-                        <ContentControl Grid.Column="1" Content="{Binding}">
-                            <ContentControl.Resources>
-                                <DataTemplate DataType="{x:Type rxui:UserError}">
-                                    <TextBlock
-                                        Text="{Binding Path=ErrorMessage}"
-                                        ToolTip="{Binding Path=ErrorCauseOrResolution}"
-                                        HorizontalAlignment="Stretch"
-                                        VerticalAlignment="Center"
-                                        TextWrapping="Wrap"
-                                        FontWeight="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type local:UserErrorMessages}},Path=ErrorMessageFontWeight}"
-                                        Margin="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type local:UserErrorMessages}},Path=MessageMargin}"/>
-                                </DataTemplate>
-                            </ContentControl.Resources>
-                        </ContentControl>
+                        <TextBlock Grid.Column="1"
```

Comment:
I had to make this change because for some reason the old way of doing it was causing a `StackOverflowException` in WPF's layouting code. This change should produce equivalent results and is in fact simpler.